task_01.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Wk 10 synthesizing."""
+
+
+def sum_orders(customers, orders):
+    """This function combines two dataset,
+        into a single dictionary.
+    Args:
+        Customer(dict)
+        orders(dict)
+    Returns:
+        Returns the combined dictionary.
+    Examples:
+            ORDERS = {1: {'customer_id': 2, 'total': 10},
+            3: {'customer_id': 2, 'total': 10},
+            4: {'customer_id': 3, 'total': 15}}
+    """
+    customers_new = {}
+    for _, order in orders.items():
```

Comment:
Whoa! I think you've made this unnecessarily complex. It could be reduced by half!